File: fix_philosophy.py
import json
import glob

for path in sorted(glob.glob('messages/*.json')):
    with open(path, 'r', encoding='utf-8') as f:
        data = json.load(f)
    
    lp = data.get('LandingPage', {})
    
    # Fix 1: remove stray colons from philosophy_intro
    if lp.get('philosophy_intro'):
        intro = lp['philosophy_intro']
        intro = intro.replace(' : ', ' ').replace(': ', ' ').replace(' :', '')
        lp['philosophy_intro'] = intro
    
    # Fix 2: add translatable philosophy_approach key
    lp['philosophy_approach'] = 'Notre approche'
    data['LandingPage'] = lp
    
    with open(path, 'w', encoding='utf-8') as f:
        json.dump(data, f, ensure_ascii=False, indent=2)
        f.write('\n')
    
    intro_preview = lp.get('philosophy_intro', '')[:70]
    print(f"Fixed {path}: {intro_preview}")

File: fix_philosophy.js
const fs = require('fs');
const path = require('path');
const dir = path.join(__dirname, 'messages');
const files = fs.readdirSync(dir).filter(f => f.endsWith('.json'));

for (const file of files) {
  const p = path.join(dir, file);
  const data = JSON.parse(fs.readFileSync(p, 'utf8'));
  if (data.LandingPage) {
    // Fix 1: remove colon from philosophy_intro (handles ": " and " :")
    if (data.LandingPage.philosophy_intro) {
      data.LandingPage.philosophy_intro = data.LandingPage.philosophy_intro
        .replace(' : ', ' ')
        .replace(': ', ' ')
        .replace(' :', '');
    }
    // Fix 2: add translatable philosophy_approach key
    data.LandingPage.philosophy_approach = 'Notre approche';
  }
  fs.writeFileSync(p, JSON.stringify(data, null, 2) + '\n');
  console.log('Updated ' + file + ' — intro: ' + data.LandingPage?.philosophy_intro?.slice(0, 60));
}
